lib/bamboozled/api/base.rb

```diff
@@ -36,9 +36,14 @@ def request(method, path, options = {})
           when 200..201
             begin
               if response.body.to_s.empty?
-                {"headers" => response.headers}.with_indifferent_access
+                {"headers" => response.headers, "code" => "200", "message" => "ok"}.with_indifferent_access
```

Comment:
Line is too long. [107/80]<br>Space inside { missing.<br>Space inside } missing.